lib/main_common.pm

```diff
@@ -568,15 +568,18 @@ sub load_jeos_tests {
     load_boot_tests();
     loadtest "jeos/firstrun";
     loadtest "jeos/record_machine_id";
+    loadtest "console/system_prepare" if is_sle;
     loadtest "console/force_scheduled_tasks";
     unless (get_var('INSTALL_LTP')) {
-        loadtest "jeos/grub2_gfxmode";
+        loadtest "jeos/grub2_gfxmode" if get_var('CONTAINER_RUNTIME');
```

Comment:
why? This is inside `INSTALL_LTP`, nothing to do with containers...

Reply:
however it is scheduled (it is actually inside _everything_ else but INSTALL_LTP. @mloviska is not available so i do not know the actual reason that it wasnt included in first place.

Reply:
What I mean is why do you run it ONLY `if get_var('CONTAINER_RUNTIME');` ?
This is wrong IMO. That should be run in all cases (except INSTALL_LTP).

Reply:
ops. you are right. will fix

Reply:
*this change is ready for review*